go/vt/servenv/http.go

```diff
@@ -46,6 +49,14 @@ func HTTPServe(l net.Listener) error {
 
 // HTTPRegisterProfile registers the default pprof HTTP endpoints with the internal servenv mux.
 func HTTPRegisterProfile() {
+	if !httpPprof {
+		return
+	}
+
+	if !pflag.Lookup("pprof-http").Changed {
```

Comment:
I'm assuming this is the case, this is `true` if the value is explicitly provided (but still the same as the default)?

Reply:
yeah: https://github.com/spf13/pflag/blob/v1.0.5/flag.go#L474-L482